src/app/beer_garden/garden.py

```diff
@@ -177,6 +180,34 @@ def create_garden(garden: Garden) -> Garden:
         The created Garden
 
     """
+    # Explicitly load default config options into garden params
+    if garden.connection_params is None:
+        garden.connection_params = {}
+
+    connection_params = getattr(garden, "connection_params", {})
+    # bg_host is required by brewtils garden spec
+    connection_params.setdefault("bg_host", "")
+
+    spec = YapconfSpec(_CONNECTION_SPEC)
+    connection_params = spec.load_config(connection_params)
```

Comment:
I ran into a problem during testing that the `connection_params` I was supplying weren't getting used at all. One problem I think you might have here is that because you are calling `load_config` before you've done the config_map ceonversion stuff, the input `connection_params` have to use the `_CONNECTION_SPEC` config names in order to get used. That's not the intended behavior.  The expected input should be "host", "port", etc. rather than "bg_host", "bg_port", and the like.  Otherwise we're effectively changing the API, which isn't our intention.

Reply:
You could tell from the unit tests that, for a reason that I can't figure out now, I was expecting `_CONNECTION_SPEC` to have the args that `create_garden` was expecting. Looking at a test POST request confirmed I had it backwards. I've pushed changes to fix this.